qiskit/aqua/operators/converters/abelian_grouper.py

```diff
@@ -83,35 +81,29 @@ def convert(self, operator: OperatorBase) -> OperatorBase:
             return operator
 
     @classmethod
-    def group_subops(cls, list_op: ListOp, fast: bool = True, use_nx: bool = False) -> ListOp:
```

Comment:
Are you concerned about backwards compatibility on this method? If someone was calling `group_subops(list_op, fast=False, use_nx=True)` their code will fail on upgrade. I get that these don't do anything now and they don't really serve a purpose since the new implementation is faster than either flag was before. 

I'm just wondering if we want to leave the flags and raise a deprecation warning that they don't do anything anymore and then we'll remove them in a future release. That would give users a chance to update their code before the kwargs without breaking code that works in 0.7.x. We wouldn't need to do anything else here since the behavior wouldn't be affected by setting the kwargs anymore, it's just about having a compatibility path for users on upgrade.

Reply:
Thank you for your feedback. Leaving the flags and raising a deprecation warning makes sense. I will do it.